docs/contributing/running-tests.md

```diff
@@ -41,3 +41,7 @@ rule will kick in and do the following:
 This works both for tasks named `test` and `latestDepTest`. But currently
 does not work for other custom test tasks, such as those created by test sets
 plugin.
+
+#### Executing single test
+
+Executing `./gradlew :instrumentation:<INSTRUMNETATION_NAME>>:test --tests <GROOVY TEST FILE NAME>` will run only the selected test.
```

Comment:
Typo: `INSTRUMNETATION_NAME` -> `INSTRUMENTATION_NAME`

Also you have one `>` too many

Reply:
Good catch!